packages/scxa-tsne-widget/src/TSnePlotWidget.js

```diff
@@ -9,85 +9,64 @@ class TSnePlotWidget extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      plotdata: {
-        metadata: [],
-        perplexities: []
-      },
+      selectedPlotType: Object.keys(this.props.defaultPlotMethodAndParameterisation)[0],
```

Comment:
You could DRY the code by extract these to a well named variable:
- this.props.defaultPlotMethodAndParameterisation
- Object.values(this.props.defaultPlotMethodAndParameterisation) by using the previously defined variable